vendor/engines/secure_rooms/db/migrate/20170405182534_add_table_token_to_card_readers.rb

```diff
@@ -0,0 +1,12 @@
+class AddTableTokenToCardReaders < ActiveRecord::Migration
+
+  class SecureRooms::CardReader < ActiveRecord::Base
+  end
+
+  def change
+    add_column :secure_rooms_card_readers, :tablet_token, :string
```

Comment:
Sure.

And I just caught that I misspelled the filename/class name. TableToken.